src/features/post_creator/actions/create_post_action.ts

```diff
@@ -2,9 +2,15 @@ import { Button } from "slack-edge";
 import { slack } from "../../../slack";
 import { sendGetStartedMessage } from "./shortcut";
 import { currentUrl } from "../../../constants";
-import { postCreatorFeatureFlag } from "..";
+import { notionEnv } from "../../../constants";
+
+/** The id of the channel where social media posts are discussed. */
+export const socialMediaChannelId =
+  notionEnv == "production" ? "C0704C5B3SQ" : "C0694MW7XJN";
```

Comment:
Even though hard-coding channel IDs is not an issue security-wise, I suggest moving this into the constants.ts file (it does not necessarily have to be an env var, though!).

Reply:
Kinda makes sense but then we need to change it for all other hardcoded channel ids in order to keep it the same everywhere. There is at least 7 or so.